docs/guides/mParticle-migration-guide.md

```diff
@@ -0,0 +1,299 @@
+4. Follow the on-screen prompts.
+
+For detailed instructions, see the documentation for the [source](/data/sources/) you want to add.
+
+## Update SDK implementation
+
+Both mParticle and Amplitude SDKs are meant to capture first party data by tracking user interactions. For the most part both work pretty similarly except some nuances around syntax. Here is high level mapping of concepts between mParticle & Amplitude.
+
+| mParticle | Amplitude | Notes                                      |
+|----------|-----------|--------------------------------------------|
+| app_key  | api_key   | Unique key to validate source of the data. |
+| Workspace | Project   | [Projects](https://help.amplitude.com/hc/en-us/articles/360058073772-Create-and-manage-organizations-and-projects#create-a-project) allow you to organize your data.  |
+| User     | User      | User who is performing action.             |
+| Identify | Identify  | [Identify](/analytics/what-is-amplitude/#user-properties-are-details-about-your-user) updates properties/attributes of the user.|
+| logEvent | Event     | [Event](/analytics/apis/http-v2-api-quickstart/) in Amplitude tracks the action user is performing.|
+| Screen   | Event     | Create an Event to track Screen views.|
+| Page     | Event     | Create an Event to track Page views.|
+|          | Group     | [Group](/guides/accounts-instrumentation-guide/) is a collection of users. In Amplitude one user could belong to multiple groups. Each group can have properties/attributes that will be available to query/forward on actions performed by any user in the group.|
+| Kits     | Plugins   | [Plugins](/data/ampli/plugin/) les you extend Amplitude by running a sequence of custom code on every event.|
+
+
+=== "Browser"
+
+    Documentation for [Browser Typescript SDK](/data/sdks/typescript-browser/).
+
+    <table>
+    <tr>
+    <td>  </td> <td> <b>mParticle</b> </td> <td> <b>Amplitude</b> </td>
+    </tr>
+    <tr>
+    <td> Identify </td>
+    <td> 
+    ```typescript
+    var identityRequest = {
+    userIdentities: { email: 'updated-email@example.com' }
+    }
+    mParticle.Identity.modify(identityRequest, identityCallback);
+    ```
+    </td>
+    <td>
+    ```typescript
+    setUserId('12091906-01011992');
+    identify(
+      Identify()
+        .set('email', 'updated-email@example.com')
+    );
+    ```
+    </td>
+    </tr>
+    <tr>
+    <td> Track </td>
+    <td>
+    ```typescript
+    mParticle.logEvent('Article Completed',
+        mParticle.EventType.EVENT-TYPE,
+        {
+            'title':'How to Create a Tracking Plan',
+            'course':'Intro to Analytics'}
+    );
+    ``` 
+    </td>
+    <td>
+    ```typescript
+    track('Article Completed', {
+      title: 'How to Create a Tracking Plan',
+      course: 'Intro to Analytics',
+    });
+    ``` 
+    </td>
+    </tr>
+    <tr>
+    <td> Group </td>
+    <td>
+    </td>
+    <td>
+    ```typescript
+    groupIdentify(
```

Comment:
I think we also include `setGroup()` here

Reply:
You mean we need to call setGroup() before every groupIdentify?